test/jdk/sun/security/tools/jarsigner/TimestampCheck.java

```diff
@@ -356,6 +357,15 @@ public static void main(String[] args) throws Throwable {
                 // sign with RSAkeysize < 1024
                 signVerbose("normal", "sign1.jar", "sign2.jar", "disabledkeysize")
                         .shouldContain("Algorithm constraints check failed on keysize")
+                        .shouldNotContain("The SHA-256 algorithm specified " +
+                            "for the -digestalg option is considered a " +
+                            "security risk and is disabled")
+                        .shouldNotContain("The SHA256withRSA algorithm " +
+                            "specified for the -sigalg option is considered " +
+                            "a security risk and is disabled")
+                        .shouldNotContain("The SHA-256 algorithm specified " +
+                            "for the -tsadigestalg option is considered a " +
+                            "security risk and is disabled")
```

Comment:
Maybe just check `.shouldNotContain("option is considered a security risk and is disabled")`?

Reply:
Ok